app/src/lib/local-storage.ts

```diff
@@ -87,3 +87,44 @@ export function getNumber(
 export function setNumber(key: string, value: number) {
   localStorage.setItem(key, value.toString())
 }
+
+/**
+ * Retrieve an array of `number` values from a given local
+ * storage entry, if found. The array will be empty if the
+ * key doesn't exist or if the values cannot be converted
+ * into numbers
+ *
+ * @param key local storage entry to read
+ */
+export function getNumberArray(key: string): ReadonlyArray<number> {
+  const numbersAsText = localStorage.getItem(key)
+  let values = new Array<number>()
+  if (numbersAsText !== null) {
+    try {
+      values = numbersAsText
+        .split(NumberArrayDelimiter)
+        .map(n => parseInt(n, 10))
```

Comment:
Given that we call this method get**Number**Array we should probably preserve the accuracy of the values. Numbers in javascript can be both floating point and integers but this method assumes integers.

Given that there's no type support specifically for integers I think we should make this method capable of handling all numbers. Also, while more academic it's possible to `.toString` an integer that won't be parseable by `parseInt`, such as `parseInt(Math.pow(10, 21).toString(), 10)`

```suggestion
        .map(parseFloat)
```